popup.js

```diff
@@ -0,0 +1,118 @@
+// Instantiate projects object
+
+const projects = [
+  {
```

Comment:
- [ ] Each object should contain at least the following information about the project: ```name```, ```description```, ```featured image```, ```technologies```, ```link to live version```, and ```link to the project source code```, you can add the links and change ```title``` to ```name``` so your code meets the requirements